sim/dex-formats.ts

```diff
@@ -641,6 +668,12 @@ export class DexFormats {
 		}
 	}
 
+	validPokemonTag(tagid: ID) {
+		const tag = Tags.hasOwnProperty(tagid) && Tags[tagid];
+		if (!tag) return false;
+		return !!(tag.speciesFilter || tag.genericFilter);
+	}
```

Comment:
Should there be a test for this? What's the point of having a property in `tags.ts` if it doesn't have one of these.

Reply:
`moveFilter` also exists. I also have `speciesNumCol` and `moveNumCol` iirc, which are planned to be used for dexsearch.